src/olympia/activity/models.py

```diff
@@ -733,13 +748,15 @@ def create(cls, action, *args, **kw):
                 ReviewActionReasonLog.objects.create(reason_id=id_, **create_kwargs)
 
         if getattr(action, 'store_ip', False):
+            ip_address = core.get_remote_addr()
```

Comment:
```suggestion
        if getattr(action, 'store_ip', False) and (ip_address := core.get_remote_addr()):
```
(and then you don't need the next change)

Reply:
https://github.com/mozilla/addons-server/pull/19616/commits/7043bce4e8ee855c1e43b9b58566f9d4b66072ea